azure/pipelines/templates/deploy.yml

```diff
@@ -243,6 +245,8 @@ jobs:
                 -dsiApiSecret "${{parameters.dsiApiSecret}}"
                 -ucasUsername "${{parameters.ucasUsername}}"
                 -ucasPassword "${{parameters.ucasPassword}}"
+                -ucasUploadBaseurl "${{parameters.ucasUploadBaseurl}}"
+                -ucasUploadFolder "${{parameters.ucasUploadFolder}}"
```

Comment:
no changes are required in `build.yml` and `release.yml` files
and the variable defined in the DevOps group can be directly referenced like
```yaml
-ucasUsername  "$(ucasUsername)"
-ucasPassword "$(ucasPassword)"
-ucasUploadBaseurl "$(ucasUploadBaseurl)"
-ucasUploadFolder "$(ucasUploadFolder)"
```

Reply:
We need to fix the docs — adding a card: https://trello.com/c/nnCGCFKY/321-update-instructions-for-adding-an-env-variable-in-azure

Reply:
Taking back changes to `build.yml` and `release.yml'.

Reply:
cool..in the deploy template file. you need to remove the parameters and just pass the variable using `$()` notation

Reply:
You mean like this? https://github.com/DFE-Digital/apply-for-teacher-training/pull/2503/commits/0fef24c1ad4ad11cc08553dc0a72312f85cf325f Should we do this for the other env vars, too?

Reply:
yes 👍..other ones need cleaning too, I will take it up.